File: C/Colleen.c
#include <stdio.h>
// 42
char *foo(void)
{
	return "#include <stdio.h>%1%c";
}
int main(void)
{
	// 42
	char	*a = foo();
	printf(a, 10, 9, 34, a);
}

File: Python/Colleen.py
#!/usr/bin/env python3

# 42 outside

def foo():
    return '#!/usr/bin/env python3\n\n# 42 outside\n\ndef foo():\n    return %r\n\ndef main():\n    # 42 inside\n\n    a = foo()\n    print(a%%a)\n\nmain()'

def main():
    # 42 inside

    a = foo()
    print(a%a)

main()
